### PR TITLE
Add Animated.Easing bindings

### PR DESCRIPTION
**Pull request by gilbert**

### CODE REVIEW

**Comment by arnarthor** [comment]
Oh wait no, @gilbert could you add the compiled JS files for this change.

File: src/animatedRe.re [hunk 1]
```diff
@@ -14,6 +14,32 @@ module CompositeAnimation = {
...
+
+  [@bs.module "react-native"] [@bs.scope "Easing"] external back : float => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external elastic : float => t = "";
+
+  [@bs.module "react-native"] [@bs.scope "Easing"] external in_ : t => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external inOut : t => t = "";
+  [@bs.module "react-native"] [@bs.scope "Easing"] external out : t => t = "";
+
+  [@bs.module "react-native"] [@bs.scope "Easing"]
+  external bezier: float => float => float => float => t = "";
```
Comment (mobily) [inline src/animatedRe.re]:
```
external bezier: (float, float, float, float) => t = "";
```

why not like this? 😉

Comment (arnarthor) [reply src/animatedRe.re]: Hmm refmt should take care of that. Maybe run refmt on the files as well

Comment (gilbert) [reply src/animatedRe.re]: I can run refmt, but the entire file changes. Is that ok?

**Comment by gilbert** [comment]
Added compiled JS and updated bezier sig.

File: src/animatedRe.re (hunk 1 above)
Comment (mobily) [reply src/animatedRe.re]: looks good! thanks :)

**Comment by mobily** [comment]
@gilbert one more thing, could you please update the `STATUS.md` file?

**Comment by gilbert** [comment]
Done

**Pull request merged**